app.js

```diff
@@ -1,183 +1,59 @@
 'use strict'
+// store hours
+var hours =['6am: ', '7am: ', '8am: ', '9am: ', '10am: ', '11am: ', '12pm: ', '1pm: ', '2pm: ', '3pm: ', '4pm: ', '5pm: ', '6pm: ', '7pm: '];
 
-var = pikeStore {  // 1st and Pike Store
-  minCust: 23;
-  maxCust: 65;
-  avgCookie: 6.3;
-  totalSales: 0;
-  hourSales: [];
+//store names
+var nameStores = [pikeStore,seaTac,seattleCenter,capHill,alki];
 
-  storeRandCust: function() {  // random number
-    return Math.random() * (this.maxCust - this.minCust) + this.minCust;
-  }
-  storeHourlySale: function (){
-    for (var i = 0; i < hours.length; i++) {
-      hours[i]
-      Math.floor(this.avgCookie * this.StoreRandCust());
-      this.hourSales.push(randCookie);
-      this.totalSales += randCookie;
-    }
-  }
-  render: function() {
-    this.storeHourlySale(); // Hourly Sales
-    var sectionEl = document.getElmentById('PikePlace');
-    sectionEl.textContent = 'Pike Place Store';
-    var ulEl = document.createElement('ul)');
-    for (var i = 0; i < hours.length; i++) {
-      var liEl = document.createElement('li');
-      liEl.textContent =hours[i] + this.hourSales[i];
-      ulEl.appendedChild(liEl);
-    }
-    var liElTotal = document.createElement('li');
-    liElTotal.textContent = 'Total Sales: ' + this.totalSales
-    ulEl.appendedChild(liElTotal);
-    sectionEl.appendedChild(ulEl);
-  }
-};
-pikeStore.render();
-
-
-var = seaTac {    // seaTac airport store
-  minCust: 3;
-  maxCust: 24;
-  avgCookie: 1.2;
-  totalSales: 0;
-  hourSales: [];
+//store Objects
 
-  storeRandCust : function() { // random number
-    return Math.random() * (this.maxCust - this.minCust) + this.minCust;
-  }
-  storeHourlySale: function() {
-    for (var i = 0; i < hours.length; i++) {
-    var randCookie = math.floor(this.avgCookie * this.storeRandCust());
-    this.hourSales.push(randCookie);
-    this.totalSales += randCookie;
-    }
+function Store(name, min, max, avgCookie) {
+  newStore.name = name;
+  newStore.min = min;
+  newStore.max = max;
+  newStore.avgCookie = avgCookie;
+  newStore.totalSales = 0;
+  newStore.hourSales = [];
+  return newStore;
```

Comment:
So for the constructor, what you want to do instead of something like 'newStore.name' is use 'this.name'. You also are not required to return anything from a constructor function.